### PR TITLE
Move isEnabled and isSchedulingEnabled  proeprties from Workflow to WorkflowBase

### PR DESCRIPTION
**Pull request by vibatra**
Move isEnabled and isSchedulingEnabled  proeprties from Workflow to WorkflowBase. This will make these properties available with WorkflowVersions as well

### CODE REVIEW

File: api-reference/beta/resources/identitygovernance-workflowbase.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ None.
 |description|String|A string that describes the purpose of the workflow.|
 |displayName|String|A string to identify the workflow. |
 |executionConditions|[microsoft.graph.identityGovernance.workflowExecutionConditions](../resources/identitygovernance-workflowexecutionconditions.md)|Defines when and for who the workflow will run.|
+|isEnabled|Boolean|If true, the `workflow` engine creates and processes `taskProcessingResults` on the users scoped to the workflow. <br><br>Supports `$filter`(`eq`,`ne`) and `orderby`.|
+|isSchedulingEnabled|Boolean|If true, the `workflow` engine executes the workflow on the schedule defined by tenant settings. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/identitygovernance-workflowbase.md:34]: Clarity on descriptions. Are these technically accurate?
```suggestion
|isEnabled|Boolean| Whether the workflow is enabled or disabled. If this setting is `true`, the workflow can be run on demand or on schedule when **isSchedulingEnabled** is `true`. <br><br>Supports `$filter`(`eq`, `ne`) and `orderBy`.|
|isSchedulingEnabled|Boolean|If `true`, the Lifecycle Workflow engine executes the workflow based on the schedule defined by tenant settings. Cannot be `true` for a disabled workflow (where **isEnabled** is `false`). |
```

File: api-reference/beta/resources/identitygovernance-workflow.md [hunk 2]
```diff
@@ -48,8 +48,6 @@ Inherits from [workflowBase](../resources/identitygovernance-workflowbase.md).
 |displayName|String|The display name of the `workflow` or `workflowVersion`. <br><br>Supports `$filter`(`eq`), `orderby`, and `$search`.|
 |executionConditions|[microsoft.graph.identityGovernance.workflowExecutionConditions](../resources/identitygovernance-workflowexecutionconditions.md)|Conditions describing when to execute the workflow and the criteria to identify in-scope subject set.|
 |id|String|Identifier used for individually addressing a specific workflow. <br><br>Supports `$filter`(`eq`).|
-|isEnabled|Boolean|If true, the `workflow` engine creates and processes `taskProcessingResults` on the users scoped to the workflow. <br><br>Supports `$filter`(`eq`,`ne`) and `orderby`.|
-|isSchedulingEnabled|Boolean|If true, the `workflow` engine executes the workflow on the schedule defined by tenant settings. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/identitygovernance-workflow.md:52]: Add back the properties with the latest descriptions. Plus append the statements "Inherited from [workflowBase](../resources/identitygovernance-workflowbase.md)."

File: changelog/Microsoft.AAD.LCM.json [hunk 3]
```diff
@@ -322,6 +322,48 @@
...
+          "Target": "workflow"
+        },
+        {
+          "Id": "a40b7de8-84cf-4012-abdf-7395b875aff5",
+          "ApiChange": "Property",
+          "ChangedApiName": "isSchedulingEnabled",
+          "ChangeType": "Deletion",
+          "Description": "Removed the `isSchedulingEnabled` property from the [workflow](https://docs.microsoft.com/en-us/graph/api/resources/workflow?view=graph-rest-beta) resource.",
+          "Target": "workflow"
+        },
```
Comment (FaithOmbongi) [inline changelog/Microsoft.AAD.LCM.json:343]: Proposing that you delete these here and make the most of the next two objects to explain this reorg.

File: changelog/Microsoft.AAD.LCM.json [hunk 4]
```diff
@@ -322,6 +322,48 @@
...
+        {
+          "Id": "a40b7de8-84cf-4012-abdf-7395b875aff5",
+          "ApiChange": "Property",
+          "ChangedApiName": "isEnabled",
+          "ChangeType": "Addition",
+          "Description": "Added the `isEnabled` property to the [workflowBase](https://docs.microsoft.com/en-us/graph/api/resources/workflowBase?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.AAD.LCM.json:349]:
```suggestion
          "Description": "Added the **isEnabled** property to the [workflowBase](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflowBase?view=graph-rest-beta) resource. [workflowBase](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflowBase?view=graph-rest-beta) is an entity type that's inherited by the following derived types in Lifecycle Workflows: [workflow](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflow?view=graph-rest-beta) and [workflowVersion](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflowVersion?view=graph-rest-beta).",
```

File: changelog/Microsoft.AAD.LCM.json [hunk 5]
```diff
@@ -322,6 +322,48 @@
...
+          "Target": "workflowBase"
+        },
+        {
+          "Id": "a40b7de8-84cf-4012-abdf-7395b875aff5",
+          "ApiChange": "Property",
+          "ChangedApiName": "isSchedulingEnabled",
+          "ChangeType": "Addition",
+          "Description": "Added the `isSchedulingEnabled` property to the [workflowBase](https://docs.microsoft.com/en-us/graph/api/resources/workflowBase?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.AAD.LCM.json:357]:
```suggestion
          "Description": "Added the **isSchedulingEnabled** property to the [workflowBase](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflowBase?view=graph-rest-beta) resource. [workflowBase](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflowBase?view=graph-rest-beta) is an entity type that's inherited by the following derived types in Lifecycle Workflows: [workflow](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflow?view=graph-rest-beta) and [workflowVersion](https://docs.microsoft.com/en-us/graph/api/resources/identitygovernance-workflowVersion?view=graph-rest-beta).",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @vibatra - Left suggestions for minor changes + the following additions:
+ Update identitygovernance-workflowversion.md as well and add the two new properties. Indicate inheritance just like you'll do for the two properties in identitygovernance-workflow.md. Add the two properties to the JSON representation as well.

File: api-reference/beta/resources/identitygovernance-workflowbase.md (hunk 1 above)
Comment (vibatra) [reply api-reference/beta/resources/identitygovernance-workflowbase.md]: This is really good feedback. I have implemented the suggested change and updated the branch/PR.

File: changelog/Microsoft.AAD.LCM.json (hunk 3 above)
Comment (vibatra) [reply changelog/Microsoft.AAD.LCM.json]: I have implemented the suggested change and updated the branch/PR.

File: changelog/Microsoft.AAD.LCM.json (hunk 4 above)
Comment (vibatra) [reply changelog/Microsoft.AAD.LCM.json]: I have implemented the suggested change and updated the branch/PR.

File: changelog/Microsoft.AAD.LCM.json (hunk 5 above)
Comment (vibatra) [reply changelog/Microsoft.AAD.LCM.json]: I have implemented the suggested change and updated the branch/PR.

File: api-reference/beta/resources/identitygovernance-workflow.md (hunk 2 above)
Comment (vibatra) [reply api-reference/beta/resources/identitygovernance-workflow.md:52]: I have implemented the suggestions and updated branch/PR.

**Review by FaithOmbongi — APPROVED** [review]
Pushed one commit to update the doc links in line with recent RUL changes.

Approved for merge.

**Comment by FaithOmbongi** [comment]
Hi @vibatra - Your schema PR was merged almost 3 weeks ago but somehow the changes are yet to reflect in both stagingBeta and beta. Any idea why? Will the changes go live before Ignite?

**Comment by vibatra** [comment]
> Hi @vibatra - Your schema PR was merged almost 3 weeks ago but somehow the changes are yet to reflect in both stagingBeta and beta. Any idea why? Will the changes go live before Ignite?

Hello Faith, we had to revert the last build due to a bug and thus had to revert the schema changed before they were pushed out to Production. I have a PR open to get the schema changes in the Thursday's deployment window.

**Pull request merged**